### ISSUE TITLE
#-163666144------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1569]
<strong>Source:</strong>`<a id="edit-button-3b290a3d8914459ca5f67f35bbf25fe1" href="/layers/3b290a3d8914459ca5f67f35bbf25fe1/edit" aria-describedby="calcite-tooltip-8ef831cc-29d3-4bec-3e2d-64e153e38a12">`

<strong>Element Location: </strong>#edit-button-3b290a3d8914459ca5f67f35bbf25fe1

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

<strong>Tags:</strong> cat.name-role-value,wcag2a,wcag412,wcag244,section508,section508.22.a,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/link-name?application=axeAPI
